test/unit/gateways/authorize_net_test.rb

```diff
@@ -626,6 +626,18 @@ def test_truncation
     end.respond_with(successful_purchase_response)
   end
 
+  def test_invalid_cvv
+    invalid_cvvs = ['47', '12345', '']
+    invalid_cvvs.each do |cvv|
+      card = credit_card(@credit_card.number, { verification_value: cvv })
+      stub_comms do
+        @gateway.purchase(@amount, card)
+      end.check_request do |endpoint, data, headers|
+        parse(data) { |doc| assert_nil doc.at_xpath('//cardCode') }
```

Comment:
I see now that that's how the other tests are working too with `parse`.  Perhaps consistency is a good thing here.  Disregard my previous nit. :)